docs/source/settingup.rst

```diff
@@ -1,16 +1,26 @@
 Setting up
 ==========
 
-Requirements:
+Requirements
+------------
 
 - DataLad
 
 Before being able to use the extension, you need to have DataLad installed, which relies on git-annex, git and python. If you don't have DataLad installed yet, please follow the instructions `here <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
```

Comment:
Idea: link git-annex (git-annex.branchable.com/) and Git (git-scm.com/) and capitalize Git and Python (both these tools are capitalized usually)

Reply:
Check!